claudiabdm/cvfy
# Add pt br lang and configure docker compose

# What was done
1. Added pt (Brazilian portuguese) language option
2. Created a small docker configuration (`Dockerfile`, `compose` and `dockerignore` files) for easy deploy
# How to test
1. You can build as always and see the new language option or
2. You can use docker to build a container
2.1. You will need docker, that you can get [here](https://docs.docker.com/engine/install/)
2.2. Just run the command `docker compose up -d` (the -d flag is optional, this makes the container run in the background)

PS: Amazing tool, thanks for developing this.
